torch/package/package_exporter.py

```diff
@@ -48,6 +48,11 @@ class _ModuleProviderAction(Enum):
     EXTERN = 2
     MOCK = 3
     DENY = 4
+    # Special case: when a module is mocked, PackageExporter writes out a
+    # `_mock` module that implements our mocking stubs. If we re-package code,
+    # we may encounter a `_mock` module from the original package. If we do,
+    # just ignore it and write a `_mock` module once.
+    REPACKAGED_MOCK_MODULE = 5
```

Comment:
would it be useful to add a repackaged_mock_modules() call like the soon-to-be-landed others? Or is that too niche of an API?